### ISSUE TITLE
Dynamic Protocol Name on Scheme Activation

### ISSUE THREAD

**Issue by mvegaca** [Microsoft/WindowsTemplateStudio#2490]
I'm working on Issue #2283 creating UserActivity templates and I've detected that when you create an app that uses scheme activation and you run it and invoke the protocol activation, Windows registers the protocol name in the app "wtsapp" for this app. If you create a new different app and you repeat the same process the protocol activation does not work because the protocol name "wtsapp" is already registered by the previous app. Even if you uninstall the first app Windows doesn't release this protocol name for scheme activation.

I think we should change the protocol name including the generating app name as a workaround, i.e. wtsApp1, wtsApp2, etc.

By this momment, Scheme activation is only avalable for new project creation, not available in right click.

What do you think about it @mrlacey @sibille?

**Comment by sibille** [comment]
I don't think we need a change here. wtsapp is a placeholder that should be replaced with sth meaningfull for the app. For testing: If you uninstall before installing the next app it works fine for me,
